src/components/Modal/styled.ts

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components';
+import { usingColors, usingTypography } from 'src/hooks/theme';
+
+export const Wrapper = styled.div`
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 700;
+  width: inherit;
+  outline: 0;
+`;
+export const Backdrop = styled.div`
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  background: rgba(0, 0, 0, 0.3);
+  z-index: 500;
+`;
+export const StyledModal = styled.div`
+  z-index: 100;
```

Comment:
Is this `z-index` necessary?

Reply:
It seems i can remove all of the `z-index` and it still works as intended. Im not sure of the best practices when it comes to z-index. I got these values from an example i found online. Should i just remove them?

Reply:
`z-index` is always tricky, and I can't say that I really understand it either. I'd keep the other `z-index`es just for good measure. If you want to get a more detailed explanation on how `z-index` works, check out this article:
https://www.joshwcomeau.com/css/stacking-contexts/